src/main/java/frc/robot/subsystems/shooter/Shooter.java

```diff
@@ -12,7 +12,9 @@ public default void enable() {}
    *
    * @param volts The volts to set. Value should be between -12.0 and 12.0.
    */
-  public default void setVoltage(double volts) {}
+  public default void runVoltage(double volts) {}
+
+  public default void runVelocity(double setPoint) {}
```

Comment:
We need a comment here specifying the units of the setPoint.  I'm assuming radians/sec?

Reply:
I updated code to specify units